libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/StepFunctionHelpers.java

```diff
@@ -41,10 +44,12 @@ public static String getIpAddress(Map<String, String> input) {
         return input.get(IP_ADDRESS);
     }
 
-    public static String getFeatureSet(Map<String, String> input) {
+    public static List<String> getFeatureSet(Map<String, String> input) {
         String featureSet = input.get(FEATURE_SET);
-        LogHelper.attachFeatureSetToLogs(featureSet);
-        return featureSet;
+        LogHelper.attachFeatureSetToLogs(Collections.singletonList(featureSet));
+        return (featureSet != null)
+                ? Stream.of(featureSet.split(",")).map(String::trim).toList()
```

Comment:
This is different to how we parse the featureset on other requests - is there a reason?